### PR TITLE
IDENTITY-5325: Fixed revoking issue of access tokens

### PR DESCRIPTION
**Pull request by GDRDABARERA**
This issue has happened due to a cache clearing problem

### CODE REVIEW

File: components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/handlers/grant/AbstractAuthorizationGrantHandler.java [hunk 1]
```diff
@@ -106,6 +106,7 @@ public OAuth2AccessTokenRespDTO issue(OAuthTokenReqMessageContext tokReqMsgCtx)
         OAuthCacheKey cacheKey = new OAuthCacheKey(cacheKeyString);
         String userStoreDomain = null;
 
+
```
Comment (madurangasiriwardena) [inline components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/handlers/grant/AbstractAuthorizationGrantHandler.java]: unnecessary new line

File: components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/TokenMgtDAO.java [hunk 2]
```diff
@@ -1722,7 +1775,8 @@ public void renameUserStoreDomainInAccessTokenTable(int tenantId, String current
     public List<AuthzCodeDO> getLatestAuthorizationCodesOfTenant(int tenantId) throws IdentityOAuth2Exception {
 
         //we do not support access token partitioning here
-        Connection connection = IdentityDatabaseUtil.getDBConnection();;
+        Connection connection = IdentityDatabaseUtil.getDBConnection();
+        ;
```
Comment (madurangasiriwardena) [inline components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/TokenMgtDAO.java]: remove this

File: components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminService.java [hunk 3]
```diff
@@ -415,14 +430,16 @@ private void updateAppAndRevokeTokensAndAuthzCodes(String consumerKey, Propertie
                     OAuthCacheKey cacheKey = new OAuthCacheKey(authorizationCode);
                     oauthCache.clearCacheEntry(cacheKey);
                 }
+
+
```
Comment (madurangasiriwardena) [inline components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminService.java]: unnecessary new line

File: components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/SQLQueries.java [hunk 4]
```diff
@@ -164,7 +164,12 @@
             "TOKEN_STATE_ID=? WHERE TOKEN_ID=?";
 
     public static final String REVOKE_ACCESS_TOKEN = "UPDATE IDN_OAUTH2_ACCESS_TOKEN SET TOKEN_STATE=?, " +
-                                                     "TOKEN_STATE_ID=? WHERE ACCESS_TOKEN=?";
+            "TOKEN_STATE_ID=? WHERE ACCESS_TOKEN=?";
+
+
```
Comment (madurangasiriwardena) [inline components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/SQLQueries.java]: unnecessary new line

File: components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/TokenMgtDAO.java [hunk 5]
```diff
@@ -40,6 +40,7 @@
 import org.wso2.carbon.identity.oauth2.util.OAuth2Util;
 import org.wso2.carbon.user.core.util.UserCoreUtil;
 
+
```
Comment (madurangasiriwardena) [inline components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/TokenMgtDAO.java]: unnecessary new line

**Comment by darshanasbg** [comment]
Squashed and resent the PR with: https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/259

**Pull request closed without merging**